.github/workflows/matplotlib-tests.yml

```
@@ -0,0 +1,75 @@
+# Run tests and integration tests on Mac, which are triggered by each master push.
+# Currently, Python3.8 is only used as an environment.
+# This is mainly for the sake of speed.
+name: matplotlib-tests
+
+on:
+  pull_request:
+    paths:
+      - tests/visualization_tests/matplotlib_tests/**
```

Matplotlib-dependent tests exist in `tests/visualization_tests/*.py` too.
```suggestion
      - tests/visualization_tests/**
```